packages/templates/usa-documentation/includes/_usa-docs-inner.twig

```diff
@@ -49,7 +49,7 @@
 
       </div>
 
-      <main class="usa-layout-docs__main desktop:grid-col-9 usa-prose usa-layout-docs" id="main-content">
+      <main class="usa-layout-docs__main desktop:grid-col-9 usa-prose" id="main-content">
```

Comment:
@amyleadem I've removed the class in 66decc0. I've also bumped this up to _potentially breaking change_ and noted in the PR description.

Mainly because:
1. This is a markup change (even if minor)
2. And because classes have been removed it _could_ impact users that have relied on these classes for custom styling.